### DISCUSSION TITLE
Add support for `show_excerpts` toggle

### DISCUSSION THREAD

**Discussion by Weiming-Hu** [mmistakes/jekyll-theme-basically-basic#58] [Ideas]
Hi,

I've added the line `show_excerpts = false` in my [_config.yml](https://github.com/Weiming-Hu/Weiming-Hu.github.io/blob/master/_config.yml) file line 92. However excerpts still show up on index pate. Could you help me with this? Thank you.

**Comment by mmistakes** [comment]
This isn't a feature of the theme. That's why `show_excerpts` has no effect. Perhaps you are confusing it with another theme?

**Comment by Weiming-Hu** [comment]
Got it. This is a feature of the minima. So I need to write my own code?

**Comment by mmistakes** [comment]
Must be something new they added since I released this theme a year ago.

I don't actively keep up on what they all do as this is something I do occasionally when I have free time.

**Comment by Weiming-Hu** [comment]
Awesome work. Thank you.

**Comment by ha0ye** [comment]
Ah, ok, I was confused about this as well, since `show_excerpts` is listed in the `quick-start guide` docs.
